pkg/handlers/generic/lifecycle/cni/cilium/strategy_helmaddon.go

```diff
@@ -1,102 +0,0 @@
-// Copyright 2023 Nutanix. All rights reserved.
```

Comment:
This was deleted because its using the common applier, correct?

Reply:
Correct.